style.css

```diff
@@ -701,3 +582,329 @@ li {
 .home_indicator {
   margin-top: 32px;
 }
+
+@media only screen and (min-width: 668px) {
+  .logo {
```

Comment:
- [ ] Great job using media queries to make your portfolio responsive for desktop devices! 💯 However, in order to target desktop screens it would be a good idea to set your breakpoint to `768px` which is the minimum width for those devices! 🤓 Please fix this one! 🛠️ 

Requirement:
![image](https://user-images.githubusercontent.com/77069011/199995812-3ec9ab16-40a4-43b8-ac15-58bdc93ac367.png)